.github/workflows/release-canary.yml

```
@@ -0,0 +1,33 @@
+name: release-canary
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  publish:
+    if: ${{ github.event.label.name == 'build' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      # Setup .npmrc file to publish to npm
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+      # Ensure that the README is published with the package
+      - run: rm -f packages/cli/README.md && cp README.md packages/cli
```

```suggestion
      - run: rm -f packages/cli/README.md && cp README.md packages/cli
      - run: echo "PR_VERSION=0.0.0-pr.${{github.event.pull_request.number}}.$(git rev-parse --short HEAD)" >> $GITHUB_ENV
```

We can write the `VERSION` to the actions env var's once, and refer to it again elsewhere, even in places where we can't `run` `git rev-parse..` for example